src/Solver/Maxwell.h

```diff
@@ -0,0 +1,85 @@
+//
+// Class Maxwell
+//   Base class for solvers for Maxwell's equations
+//
```

Comment:
Also here we should mention the original paper Mithra !?

Reply:
I would cite Mithra in the class FDTDSolver, not in the base Maxwell class (as the FEM solver will also inherit from this).